Browser/keywords/interaction.py

```diff
@@ -312,6 +312,7 @@ def click(self, selector: str, button: MouseButton = MouseButton.left):
 
         [https://forum.robotframework.org/t//4238|Comment >>]
         """
+        logger.info(f"Clicks the element '{selector}'.")
```

Comment:
I think this logging should go inside of `click_with_options` method. Then it would be same for both keywords.

Reply:
Done. It could of course log more if it's there but...